### DISCUSSION TITLE
Release 2.9.3 + (short term) future thoughts

### DISCUSSION THREAD

**Discussion by ChrisMaunder** [codeproject/CodeProject.AI-Server#188] [Announcements]
I've pushed 2.9.3 to our CDN, and [download links are here](https://codeproject.github.io). I've not made this the "latest" since there are a number of non-trivial changes, the least of which is a new hosting provider and a different method (under the hood) of checking for updates to the server and to modules. I apologise deeply, but there will be some teething issues, but I will work hard to smooth things out quickly.

Future plans at this point centre around modernising some of the frameworks. We stuck to Python 3.7 for many modules simply as a cut-and-paste of the original code that went into the modules, but in many cases we should all look forward not backwards and ensure we're on the latest versions of runtimes for performance, security and lifespan.

**Comment by indi1984** [comment]
Happy to report that the install was flawless, and the YOLOv5 6.2 GPU detection seems to be working normally.   Up to date Win 10, NVidia GTX970, Driver version 566.14, CUDA 11.8.89, Compute 5.2, cuDNN 8.5.   Thank you Chris and team for the hard work.

**Comment by ChrisMaunder** [comment]
Excellent! Thanks for the update.
